components/alibi-explain-server/alibiexplainer/anchor_images.py

```diff
@@ -38,10 +42,13 @@ def __init__(
         if explainer is None:
             raise Exception("Anchor images requires a built explainer")
         self.anchors_image = explainer
+        if EXPLAIN_RANDOM_SEED == "True" and EXPLAIN_RANDOM_SEED_VALUE.isdigit():
+            self.seed = int(EXPLAIN_RANDOM_SEED_VALUE)
         self.kwargs = kwargs
 
     def explain(self, inputs: List) -> Explanation:
-        np.random.seed(0)
+        if self.seed:
```

Comment:
Is it possible that the `self.seed`  is never set and the if statement will throw an error? 
Wouldn't an approach based on `hasattr`  be more appropriate?
Same for the other variants of anchor.

Reply:
Fixed in #4078